### PR TITLE
Split shortlist and evaluation criteria into multiple sections

### PR DESCRIPTION
**Pull request by TheDoubleK**
By having multiple sections with questions depending only on a single lot we can have different text on the "parent" pages for each section.

### CODE REVIEW

**Comment by allait** [comment]
👍

**Comment by pcraig3** [comment]
Small bug with the markdown, but everything else looks good.

screen shot 2016-04-19 at 18 51 04

File: frameworks/digital-outcomes-and-specialists/manifests/edit_brief.yml [hunk 1]
```diff
@@ -55,56 +55,129 @@
...
+  description: |
+    After you’ve published your requirements, specialists will apply and tell you:
+    
+    - when they can start work
+    - what their day rate is
+    - if they have the essential skills and experience
+    - which nice-to-have skills and experience they have
+    
     You can shortlist the specialists who:
-
```
Comment (pcraig3) [inline frameworks/digital-outcomes-and-specialists/manifests/edit_brief.yml]: `\n`

File: frameworks/digital-outcomes-and-specialists/manifests/edit_brief.yml [hunk 2]
```diff
@@ -55,56 +55,129 @@
...
     - can start work when you need them to
+    - are within your budget
     - meet all the essential skills and experience
     - provide the most nice-to-have skills and experience
 
-    [Read about how to shortlist.](http://giphy.com/)
+    [Read about how to shortlist.](https://www.gov.uk/guidance/how-to-shortlist-digital-outcomes-and-specialists-suppliers)
```
Comment (pcraig3) [inline frameworks/digital-outcomes-and-specialists/manifests/edit_brief.yml]: /giphy lol

File: frameworks/digital-outcomes-and-specialists/manifests/edit_brief.yml [hunk 3]
```diff
@@ -55,56 +55,129 @@
...
   step: 2
   description: |
-    You need to tell suppliers how you’re going to evaluate them when you publish your requirements.
+    After you’ve published your requirements, suppliers will apply and tell you:
+    
+    - if they have the essential skills and experience
+    - which nice-to-have skills and experience they have
+    
+    You can shortlist the suppliers who:
+    - meet all the essential skills and experience
```
Comment (pcraig3) [inline frameworks/digital-outcomes-and-specialists/manifests/edit_brief.yml]: `\n`

**Comment by pcraig3** [comment]
(Can confirm it fixes the duplicated weightings questions though!)

File: frameworks/digital-outcomes-and-specialists/manifests/edit_brief.yml (hunk 2 above)
Comment (TheDoubleK) [inline frameworks/digital-outcomes-and-specialists/manifests/edit_brief.yml]: Surely that was one of yours?

**Pull request merged**